systems/plants/@RigidBodyManipulator/doKinematics.m

```diff
@@ -337,10 +337,11 @@
 ret{1} = zeros(16, nq);
 for i = 2 : nb
   body = bodies(i);
-  T_body_to_parent = T{body.parent} \ T{i};
+  %T_body_to_parent = T{body.parent} \ T{i};
+  T_body_to_parent = T{body.parent}^-1 * T{i};
```

Comment:
but it doesn't support ^-1 either, does it? In any case, the results can't be right in general, as you're outside of the class of polynomials.